_minutes/README.md

```diff
@@ -1,21 +1,20 @@
 # Meetings
 
-The [WebExtensions Community group](https://www.w3.org/community/webextensions/) meets virtually every other week, for one hour.
-The instructions to join the meeting and agenda are available at https://www.w3.org/events/meetings/7fc25ca5-a50c-498c-82e5-f48fc96e1637.
+The [WebExtensions Community group](https://www.w3.org/community/webextensions/) meets virtually every other week, for one hour. The instructions to join the meeting and agenda are available at https://www.w3.org/events/meetings/7fc25ca5-a50c-498c-82e5-f48fc96e1637.
```

Comment:
Why did you merge them on one line?

If you want to change this line anyway, let's change the link to the one from https://github.com/w3c/webextensions/pull/129/files (and then I can close that PR)

Reply:
Since this is a trivial change I'll force-push the whitespace change to the branch and then merge the PR